server.py

```diff
@@ -19,16 +19,17 @@
         output_sender = NetworkTerminalOutput(client)
         input_sender = NetworkTerminalInput(client)
         with client:
-            # gm = GameManager(scenes_list, output_sender, input_sender)
+            gm = GameManager(scenes_list, output_sender, input_sender)
             server_output.output(f"connection made from {address}")
             # client.settimeout(15)
-            while True:
-                # gm.start(intro_scene)
-                data = input_sender.input("enter a word: ")
+            # while True:
+            gm.start(intro_scene)
```

Comment:
Wouldn't it start the same scene each time?

Reply:
no, because `gamemanager.start()` uses the `.get_next_scene()` method
   
` def start(self, current_scene):`
        `self.current_scene = current_scene`
       `while True:`
            `next_scene_name = self.run_scene()`
           `self.current_scene = self.get_next_scene(next_scene_name)`

Reply:
Ok, so why do you have the while True before the `GameManager` ? are you planning of serving multiple clients in parallel?